src/modelbuilder_base.nit

```diff
@@ -472,3 +506,38 @@ redef class ADoc
 		return res
 	end
 end
+
+redef class AQclassid
+	# The name of the package part, if any
+	fun mpackname: nullable String do
+		var nqualified = n_qualified
+		if nqualified == null then return null
+		var nids = nqualified.n_id
+		if nids.length <= 0 then return null
+		return nids[0].text
+	end
+
+	# The name of the module part, if any
+	fun mmodname: nullable String do
+		var nqualified = n_qualified
+		if nqualified == null then return null
+		var nids = nqualified.n_id
+		if nids.length <= 1 then return null
+		return nids[1].text
+	end
+
+	# Does the given mclass match the full qualified name?
```

Comment:
Instead of `Does the given mclass match...` you should use the format 

~~~
Does `mclass` match...
~~~

It's more concise and clearer.